### PR TITLE
Parametrized missing options on templates'

### PR DESCRIPTION
**Pull request by cohandv**

### CODE REVIEW

File: deployments/k8s/helm/signalfx-agent/values.yaml [hunk 1]
```diff
@@ -67,6 +67,9 @@ minReadySeconds:
 # Namespace to deploy agent in (Optional: Will default to release namespace)
 namespace:
 
+# For Troubleshooting if you do not want to bind to host change
```
Comment (keitwb) [inline deployments/k8s/helm/signalfx-agent/values.yaml:70]:
```suggestion
# Set the DNS policy for the agent daemonset/deployment pods
```

File: deployments/k8s/helm/signalfx-agent/templates/deployment.yaml [hunk 2]
```diff
@@ -37,7 +37,7 @@ spec:
       annotations:
         {{- toYaml .Values.podAnnotations | trim | nindent 8 }}
     spec:
-      dnsPolicy: ClusterFirstWithHostNet
+      dnsPolicy: {{ .Values.dnsPolicy }}
```
Comment (keitwb) [inline deployments/k8s/helm/signalfx-agent/templates/deployment.yaml:40]: Do this same thing for the DaemonSet too so it is consistent.

**Comment by emaderer** [comment]
Hey @cohandv , what's the status of this PR?

File: deployments/k8s/helm/signalfx-agent/templates/deployment.yaml (hunk 2 above)
Comment (cohandv) [reply deployments/k8s/helm/signalfx-agent/templates/deployment.yaml:40]: sorry the delay, I changed it to use the value

Comment (emaderer) [reply deployments/k8s/helm/signalfx-agent/templates/deployment.yaml:40]: Thanks! @keitwb  mind taking another look?

Comment (keitwb) [reply deployments/k8s/helm/signalfx-agent/templates/deployment.yaml:40]: I don't  see the change in the daemonset.

Comment (emaderer) [reply deployments/k8s/helm/signalfx-agent/templates/deployment.yaml:40]: @cohandv  ^^

**Pull request closed without merging**